apps/passport-client/src/zapp/collections.ts

```diff
@@ -21,8 +30,16 @@ export function getPODsForCollections(
     .flatMap((collectionId) =>
       collectionId === "Devcon SEA"
         ? pcds.getAllPCDsInFolder("Devcon SEA")
+        : collectionId === "Email"
```

Comment:
and then refactor this into a nice function if collection id is in VIRTUAL_COLLECTIONS